src/INode.java

```diff
@@ -431,10 +451,24 @@ private ASTValue evalQueueWait(WordsEnvironment environment) {
 		throw new AssertionError("Not yet implemented");
 	}
 
-	private ASTValue evalReferenceList(WordsEnvironment environment) {
-		// TODO
-		//throw new AssertionError("Not yet implemented");
-		return new ASTValue(ValueType.NOTHING);
+	private ASTValue evalReferenceList(WordsEnvironment environment) throws WordsRuntimeException {
+		String firstObjRefName = children.get(0).eval(environment).stringValue;
+		WordsObject currentObj = environment.getObject(firstObjRefName);
+		
+		for (int i = 1; i < children.size(); i++) {
+			String propertyRefName = children.get(i).eval(environment).stringValue;
+			WordsProperty prop = currentObj.getProperty(propertyRefName);
+			if (prop.type != WordsProperty.PropertyType.OBJECT) {
+				return new ASTValue(ValueType.NOTHING);
```

Comment:
I think this should be an exception, not NOTHING.  A reference list is a chain of objects, the first provided by name, the others provided as property names.  At every step (including the last), it must be an object.  Otherwise, the user would be trying to dereference a non-object, which is an error on their part.